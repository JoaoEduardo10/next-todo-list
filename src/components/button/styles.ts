import styled, { css } from 'styled-components';

export const Button = styled.button`
  ${({ theme }) => css`
    margin-top: 3rem;
    width: 16rem;
    padding: ${theme.padding.small};
    background-color: ${theme.colors.purpleColor};
    border: none;
    border-radius: 1.6rem;
    font-size: 1.8rem;
    line-height: 100%;
    color: ${theme.colors.primaryColor};
    cursor: pointer;
    transition: all 300ms ease-in-out;

    &:hover {
      transform: scale(1.1);
    }
    &:disabled {
      background-color: ${theme.colors.secondaryColor};
      cursor: not-allowed;
      color: ${theme.colors.blackColor};
      transform: scale(1);
    }
  `}
`;
