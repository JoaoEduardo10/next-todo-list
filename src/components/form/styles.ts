import styled, { css } from 'styled-components';

export const Conteiner = styled.form`
  ${({ theme }) => css`
    background-color: ${theme.colors.primaryColor};
    width: 40%;
    height: 90%;
    padding: ${theme.padding.extra_big};
    display: flex;
    flex-direction: column;
    border-radius: 0.5rem;
    box-shadow: 0.1rem 0.1rem 0.9rem ${theme.colors.secondaryColor};
    align-items: center;
  `}
`;
