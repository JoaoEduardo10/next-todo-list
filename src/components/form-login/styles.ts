import styled, { css } from 'styled-components';

import { Label } from '../inpult/styles';
import { Button } from '../button/styles';

export const Form = styled.form`
  ${({ theme }) => css`
    background-color: ${theme.colors.secondaryColor};
    width: 40%;
    height: 95%;
    padding: ${theme.padding.large};
    display: flex;
    flex-direction: column;
    gap: 4rem;
    border-radius: 0.5rem;
    box-shadow: 0.1rem 0.1rem 0.2rem ${theme.colors.primaryColor};
    align-items: center;

    ${theme.media_screen_size.phone} {
      width: 90%;
      padding: ${theme.padding.small};
      justify-content: center;

      ${Label} {
        width: 100%;
      }
    }

    ${theme.media_screen_size.tablet} {
      justify-content: center;
      width: 80%;
      padding: 0 ${theme.padding.large};

      ${Label} {
        height: 10rem;
        margin-bottom: 2.4rem;

        & input {
          font-size: 3.6rem;
        }

        & svg {
          font-size: 4.4rem;
        }
      }

      ${Button} {
        width: 50%;
        height: 10rem;
        font-size: 4rem;
      }

      a {
        font-size: 3rem;
      }
    }

    ${theme.media_screen_size.lapTop} {
      justify-content: center;

      ${Label}:nth-child(3) {
        margin-bottom: 0.9;
      }

      ${Label} {
        height: 7rem;
        width: 100%;

        & input {
          font-size: 2.6rem;
        }

        & svg {
          font-size: 3.4rem;
        }
      }

      ${Button} {
        width: 90%;
        height: 6rem;
        font-size: 3rem;
      }

      a {
        font-size: 2.4rem;
      }
    }

    & a {
      color: ${theme.colors.blackColor};
      text-decoration: none;
      padding: ${theme.padding.small};
      transition: all 300ms ease-in-out;

      &:hover {
        color: ${theme.colors.purpleColor};
        text-decoration: underline;
      }
    }
  `}
`;

export const Heading = styled.h1`
  ${({ theme }) => css`
    margin-bottom: 6rem;
    color: ${theme.colors.purpleColor};
    font-family: Arial, Helvetica, sans-serif;
    font-size: 3.8rem;

    ${theme.media_screen_size.phone} {
      margin-bottom: 4rem;
    }

    ${theme.media_screen_size.tablet} {
      font-size: 8rem;
    }

    ${theme.media_screen_size.lapTop} {
      font-size: 4rem;
      margin-bottom: 1.6rem;
    }
  `}
`;
