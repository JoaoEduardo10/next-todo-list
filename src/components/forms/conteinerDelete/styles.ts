import styled, { css } from 'styled-components';
import { Button } from '../../button/styles';

const showDelete = (show: boolean, rendered: boolean) => css`
  ${show
    ? css`
        animation: openShow 300ms ease-in-out;
        opacity: 1;

        @keyframes openShow {
          0% {
            transform: translateX(100%);
            opacity: 0;
          }

          100% {
            transform: translate(0);
          }
        }
      `
    : css`
        visibility: hidden;
        animation: closeShow 300ms ease-in-out;
        opacity: 0;

        @keyframes closeShow {
          0% {
            transform: translateX(0);
            opacity: ${rendered ? '1' : '0'};
            visibility: visible;
          }

          100% {
            transform: translate(-100%);
            visibility: hidden;
          }
        }
      `}
`;

export const Conteiner = styled.div<{ show: boolean; rendered: boolean }>`
  ${({ show, rendered }) => css`
    position: fixed;
    top: 0%;
    left: 0%;
    height: 100%;
    background-color: rgba(0, 0, 0, 0.6);
    display: flex;
    justify-content: center;
    align-items: center;

    ${showDelete(show, rendered)}
  `}
`;

export const ConteinerDelete = styled.section`
  ${({ theme }) => css`
    background-color: ${theme.colors.primaryColor};
    width: 40%;
    border-radius: 0.5rem;
    padding: ${theme.padding.extra_small};
    display: flex;
    flex-direction: column;
    justify-content: space-between;
  `}
`;

export const Heading = styled.h3`
  ${({ theme }) => css`
    color: ${theme.colors.redColor};
    font-weight: bold;
    margin-bottom: 1.6rem;
  `}
`;

export const Paragraph = styled.p`
  ${() => css`
    color: gray;
  `}
`;

export const ConteinerButton = styled.div`
  ${({ theme }) => css`
    display: flex;
    width: 50%;
    gap: 1.6rem;

    ${Button} {
      width: 100%;
      border-radius: 5rem;
      transition: 300ms ease-in-out;

      &:nth-child(1) {
        background-color: ${theme.colors.redColor};
        transition: 300ms ease-in-out;

        &:hover {
          background-color: transparent;
          border: 0.2rem solid ${theme.colors.redColor};
          color: ${theme.colors.redColor};
        }
      }

      &:nth-child(2) {
        background-color: ${theme.colors.secondaryColor};
        color: ${theme.colors.purpleColor};

        &:hover {
          border: 0.2rem solid ${theme.colors.purpleColor};
        }
      }

      &:hover {
        transform: scale(1);
      }
    }
  `}
`;
