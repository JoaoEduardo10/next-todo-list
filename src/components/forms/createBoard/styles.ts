import styled, { css } from 'styled-components';
import { Button } from '../../button/styles';
import { Label } from '../../inpult/styles';

const ShowCreateBoard = (rendering: boolean, show: boolean) => css`
  ${show
    ? css`
        animation: openShow 300ms ease-in-out;
        opacity: 1;

        @keyframes openShow {
          0% {
            transform: translateX(100%);
            opacity: 0;
          }

          100% {
            transform: translate(0);
          }
        }
      `
    : css`
        visibility: hidden;
        animation: closeShow 300ms ease-in-out;
        opacity: 0;

        @keyframes closeShow {
          0% {
            transform: translateX(0);
            opacity: ${rendering ? '1' : '0'};
            visibility: visible;
          }

          100% {
            transform: translate(-100%);
            visibility: hidden;
          }
        }
      `}
`;

export const ShowCreateBoardConteiner = styled.div<{
  rendering: boolean;
  show: boolean;
}>`
  ${({ theme, rendering, show }) => css`
    position: fixed;
    top: 0%;
    width: 100%;
    height: 100vh;
    background-color: rgba(0, 0, 0, 0.7);
    display: flex;
    justify-content: center;
    align-items: center;
    z-index: 10;

    ${ShowCreateBoard(rendering, show)}
  `}
`;

export const FormCreateBoard = styled.form`
  ${({ theme }) => css`
    position: relative;
    background-color: ${theme.colors.primaryColor};
    height: 40%;
    width: 50%;
    display: flex;
    flex-direction: column;
    padding: ${theme.padding.small};
    gap: 1.6rem;
    align-items: center;
    justify-content: center;
    border-radius: 0.5rem;

    & > svg {
      position: absolute;
      top: 10%;
      right: 5%;
      font-size: 2.5rem;
      color: ${theme.colors.purpleColor};
      cursor: pointer;
    }

    ${Button} {
      width: 50%;
    }

    ${Label} {
      width: 80%;
    }
  `}
`;
