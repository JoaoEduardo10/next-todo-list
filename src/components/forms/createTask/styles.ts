import styled, { css } from 'styled-components';

import { Label as ConteinerLabel } from '../../inpult/styles';

const ShowTask = (rendering: boolean, show: boolean) => css`
  ${show
    ? css`
        animation: openShow 300ms ease-in-out;
        opacity: 1;

        @keyframes openShow {
          0% {
            transform: translateX(100%);
            opacity: 0;
          }

          100% {
            transform: translate(0);
          }
        }
      `
    : css`
        visibility: hidden;
        animation: closeShow 300ms ease-in-out;
        opacity: 0;

        @keyframes closeShow {
          0% {
            transform: translateX(0);
            opacity: ${rendering ? '1' : '0'};
            visibility: visible;
          }

          100% {
            transform: translate(-100%);
            visibility: hidden;
          }
        }
      `}
`;

export const ConteinerCreateTask = styled.div<{
  show: boolean;
  rendering: boolean;
}>`
  ${({ show, rendering }) => css`
    position: fixed;
    top: 0%;
    left: 0%;
    width: 100%;
    height: 100vh;
    background-color: rgba(0, 0, 0, 0.7);
    display: flex;
    justify-content: center;
    align-items: center;
    z-index: 25;

    ${ShowTask(rendering, show)}
  `}
`;

export const CreateTaskForm = styled.form`
  ${({ theme }) => css`
    width: 40%;
    height: 90%;
    border-radius: 0.5rem;
    background-color: ${theme.colors.primaryColor};
    padding: ${theme.padding.extra_small};
    display: flex;
    flex-direction: column;
    overflow-y: auto;
    align-items: center;

    ${ConteinerLabel} {
      box-shadow: none;
      border: 0.1rem solid #222;
    }

    ${theme.media_screen_size.phone} {
      width: 95%;
    }
  `}
`;

export const Heading = styled.h2`
  ${({ theme }) => css`
    display: flex;
    justify-content: space-between;
    align-items: center;
    width: 100%;

    & span {
      color: ${theme.colors.lite_greenColor};
      display: inline-block;
      font-size: 3.5rem;
      transform: translateY(13%);
    }
  `}
`;

export const ConteinerInput = styled.div`
  ${({ theme }) => css`
    display: flex;
    flex-direction: column;
    width: 100%;
    margin: 1.6rem 0;
  `}
`;

export const Label = styled.label`
  ${({ theme }) => css`
    margin-bottom: 1rem;
  `}
`;

export const TextArea = styled.textarea`
  ${({ theme }) => css`
    height: 20rem;
    outline: none;
    border: 0.1rem solid #222;
    padding: 0.6rem;
  `}
`;
