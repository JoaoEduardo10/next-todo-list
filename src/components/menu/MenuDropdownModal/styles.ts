import styled, { css, DefaultTheme } from 'styled-components';
import { Button } from '../../button/styles';

const showMenu = (show: boolean, rendering: boolean) => css`
  ${show &&
  css`
    animation: showMenu 300ms ease-in-out;
    opacity: 1;

    @keyframes showMenu {
      0% {
        transform: translateX(-100%);
      }

      100% {
        transform: translateX(0);
      }
    }
  `}

  ${!show &&
  css`
    animation: hiddenShow 300ms ease-in-out;

    @keyframes hiddenShow {
      0% {
        transform: translateX(0);
        ${rendering ? 'opacity: 1;' : 'opacity: 0;'}
      }

      100% {
        transform: translateX(-100%);
        opacity: 0;
      }
    }
  `}
`;

const boardClicled = (
  board: string,
  boardId: string,
  theme: DefaultTheme,
) => css`
  ${board == boardId &&
  `
    background-color: ${theme.colors.purpleColor};
    transition: all 300ms ease-in-out;

    span {
      color: ${theme.colors.primaryColor};
    }
    `}
`;

export const MenuMobile = styled.div<{ show: boolean; rendering: boolean }>`
  ${({ theme, show, rendering }) => css`
    display: none;
    background-color: rgba(0, 0, 0, 0.7);
    width: 100%;
    height: 100vh;

    ${theme.media_screen_size.phone} {
      display: flex;
      justify-content: center;
      align-items: center;
      opacity: 0;

      ${showMenu(show, rendering)}
    }
  `}
`;

export const Menu = styled.ul`
  ${({ theme }) => css`
    height: 80%;
    width: 80%;
    background-color: ${theme.colors.primaryColor};
    transform: translateY(-7%);
    overflow-y: auto;
    display: flex;
    flex-direction: column;

    h2 {
      margin: 2rem 0;
      padding: ${theme.padding.small};
    }
  `}
`;

export const Li = styled.li<{ board: string; boardId: string }>`
  ${({ theme, board, boardId }) => css`
    margin-bottom: ${theme.padding.small};
    display: flex;
    align-items: center;
    justify-content: center;
    gap: 1rem;
    border-radius: 0 100px 100px 0;
    width: 90%;
    padding: 15px 24px;
    background-color: transparent;

    svg {
      font-size: 3rem;
      transform: translateY(8%);
    }

    span {
      font-size: 1.7rem;
      font-weight: bold;
      color: gray;
    }

    ${boardId.length > 1 && boardClicled(board, boardId, theme)}
  `}
`;

export const ConteinerDiv = styled.div`
  ${({ theme }) => css`
    flex: 1;
    width: 100%;
    height: 100%;
    display: flex;
    justify-content: center;
    padding: ${theme.padding.small};

    ${Button} {
      height: 10rem;
      width: 100%;
    }
  `}
`;
