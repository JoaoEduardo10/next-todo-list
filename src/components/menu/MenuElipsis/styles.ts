import styled, { css } from 'styled-components';
import { Button } from '../../button/styles';

const handleShow = (show: boolean, rendered: boolean) => css`
  ${show
    ? css`
        opacity: 1;
        animation: openShow 300ms ease-in-out;

        @keyframes openShow {
          0% {
            transform: translateX(100%);
            opacity: 0;
          }

          100% {
            transform: translate(0);
          }
        }
      `
    : css`
        opacity: 0;
        animation: closeShow 0.5s ease-in-out;

        @keyframes closeShow {
          0% {
            transform: translateX(0);
            opacity: ${rendered ? '1' : '0'};
          }

          25% {
            transform: translateX(-40%);
          }

          100% {
            transform: translate(100%);
          }
        }
      `}
`;

export const Conteiner = styled.div<{ show: boolean; rendered: boolean }>`
  ${({ theme, show, rendered }) => css`
    position: fixed;
    right: 2%;
    top: 15%;
    width: 15.5rem;
    background-color: ${theme.colors.primaryColor};
    padding: ${theme.padding.small};
    display: flex;
    flex-direction: column;
    gap: 1rem;
    border-radius: 0.5rem;
    border: 0.2rem solid ${theme.colors.secondaryColor};
    box-shadow: 0rem 0rem 4.9rem ${theme.colors.blackColor};
    opacity: 0;

    & div {
      width: 100%;

      & ${Button} {
        width: 100%;
        height: 100%;
      }
    }

    ${handleShow(show, rendered)}
  `}
`;

export const Paragraph = styled.p`
  ${({ theme }) => css`
    color: gray;
    cursor: pointer;
    transition: all 300ms ease-in-out;

    &:nth-child(2) {
      color: ${theme.colors.redColor};
    }

    &:hover {
      transform: scale(1.1);
    }
  `}
`;
