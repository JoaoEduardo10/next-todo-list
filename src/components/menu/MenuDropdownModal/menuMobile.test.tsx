import { fireEvent, screen, act } from '@testing-library/react';
import { MenuDropdownModal } from '.';
import { theme } from '../../../styles/theme';
import { renderTheme } from '../../../utils/render-theme';
import configureStore from 'redux-mock-store';

const mockStore = configureStore([]);
jest.useFakeTimers();

describe('MenuDropdownModal component', () => {
  let store: any;

  beforeEach(() => {
    store = mockStore({
      boards: [
        { id: 'board1', boardName: 'Board 1' },
        { id: 'board2', boardName: 'Board 2' },
      ],
    });
  });

  afterEach(() => {
    jest.clearAllMocks();
  });

  it('should redender a mobile menu with links', () => {
    renderTheme(<MenuDropdownModal show={true} />, store);

    const menu = screen.getByLabelText('Menu');
    const links = screen.getAllByLabelText('Link');

    expect(menu).toBeInTheDocument();
    expect(links.length).toBe(2);
  });

  it('should would render the first link with being active', () => {
    renderTheme(<MenuDropdownModal show={true} />, store);

    const links = screen.getAllByLabelText('Link');

    act(() => {
      jest.advanceTimersByTime(500);
    });

    expect(links[0]).toHaveStyle({
      'background-color': theme.colors.purpleColor,
    });
    expect(links[1]).toHaveStyle({
      'background-color': 'transparent',
    });
  });

  it('should make the link you click active', () => {
    renderTheme(<MenuDropdownModal show={true} />, store);

    const links = screen.getAllByLabelText('Link');

    act(() => {
      jest.advanceTimersByTime(500);
    });

    fireEvent.click(links[1]);

    expect(links[0]).toHaveStyle({
      'background-color': 'transparent',
    });

    expect(links[1]).toHaveStyle({
      'background-color': theme.colors.purpleColor,
    });
  });

  it('should redefine the buttao to create a board', () => {
    const newStore = mockStore({
      boards: [],
    });

    renderTheme(<MenuDropdownModal show={true} />, newStore);

    const conteinerDivButton = screen.getByLabelText('Conteiner Button');

    act(() => {
      jest.advanceTimersByTime(500);
    });

    expect(conteinerDivButton).toBeInTheDocument();
  });

  it('should not render a MenuDropdownModal', () => {
    const newStore = mockStore({
      boards: [],
    });

    renderTheme(<MenuDropdownModal show={false} />, newStore);

    const menu = screen.getByLabelText('Menu');

    act(() => {
      jest.advanceTimersByTime(500);
    });
  });

  it('should to match snapshot', () => {
    renderTheme(<MenuDropdownModal show={true} />, store);

    const menu = screen.getByLabelText('Menu');

    expect(menu).toMatchSnapshot();
  });
});
