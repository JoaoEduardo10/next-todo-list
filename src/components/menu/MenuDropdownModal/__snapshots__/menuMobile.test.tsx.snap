// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuDropdownModal component should to match snapshot 1`] = `
.c0 {
  display: none;
  background-color: rgba(0,0,0,0.7);
  width: 100%;
  height: 100vh;
}

.c1 {
  height: 80%;
  width: 80%;
  background-color: #FFFFFF;
  -webkit-transform: translateY(-7%);
  -ms-transform: translateY(-7%);
  transform: translateY(-7%);
  overflow-y: auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c1 h2 {
  margin: 2rem 0;
  padding: 1.6rem;
}

.c3 {
  margin-bottom: 1.6rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  gap: 1rem;
  border-radius: 0 100px 100px 0;
  width: 90%;
  padding: 15px 24px;
  background-color: transparent;
}

.c3 svg {
  font-size: 3rem;
  -webkit-transform: translateY(8%);
  -ms-transform: translateY(8%);
  transform: translateY(8%);
}

.c3 span {
  font-size: 1.7rem;
  font-weight: bold;
  color: gray;
}

.c2 {
  margin-bottom: 1.6rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  gap: 1rem;
  border-radius: 0 100px 100px 0;
  width: 90%;
  padding: 15px 24px;
  background-color: transparent;
  background-color: #827FD3;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c2 svg {
  font-size: 3rem;
  -webkit-transform: translateY(8%);
  -ms-transform: translateY(8%);
  transform: translateY(8%);
}

.c2 span {
  font-size: 1.7rem;
  font-weight: bold;
  color: gray;
}

.c2 span {
  color: #FFFFFF;
}

@media (max-width:780px) {
  .c0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    opacity: 0;
    -webkit-animation: showMenu 300ms ease-in-out;
    animation: showMenu 300ms ease-in-out;
    opacity: 1;
  }
}

<div
  aria-label="Menu"
  class="c0"
>
  <ul
    class="c1"
  >
    <h2>
      Todos os quadros (
      2
      )
    </h2>
    <li
      aria-label="Link"
      class="c2"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM5 19V5h6v14H5zm14 0h-6v-7h6v7zm0-9h-6V5h6v5z"
        />
      </svg>
       
      <span>
        Board 1
      </span>
    </li>
    <li
      aria-label="Link"
      class="c3"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM5 19V5h6v14H5zm14 0h-6v-7h6v7zm0-9h-6V5h6v5z"
        />
      </svg>
       
      <span>
        Board 2
      </span>
    </li>
  </ul>
</div>
`;
